components/Slider/src/MDCSlider.m

```diff
@@ -209,6 +213,24 @@ - (UIColor *)backgroundTrackTickColorForState:(UIControlState)state {
   return color;
 }
 
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state {
+  _thumbRadiusesForState[@(state)] = [NSNumber numberWithFloat:thumbRadius];
+  if (self.state == state) {
+    [self updateThumbForState];
+  }
+}
+
+- (CGFloat)thumbRadiusForState:(UIControlState)state {
+  if ([_thumbRadiusesForState objectForKey:@(state)]) {
+    return [_thumbRadiusesForState[@(state)] floatValue];
+  }
+  CGFloat radius = 0.f;
```

Comment:
Can we use the default value instead of 0? I don't think we want clients to be "hiding" the thumb by making it too small to render.

Reply:
Done